### DISCUSSION TITLE
整合three.js实现？

### DISCUSSION THREAD

**Discussion by kireiyume** [zouyaoji/vue-cesium#547] [General]
请问作者，目前的情况来说vue-cesium能否进行整合three.js，这样就能继续组件化开发了。不知道能否指点下？谢谢

**Comment by zouyaoji** [comment]
可以啊，你最好能给一个cesiumjs和threejs联合用的例子，然后我就可以给你说vue-cesium咋集成了。

比如下面这样：
https://github.com/meteosci/vue-maplibre/blob/dev/packages/shared/layer/GLTFLayer.ts

https://github.com/meteosci/vue-maplibre/blob/dev/packages/components/layer/gltf/index.ts
